### PR TITLE
Fix the local debug since the tag changed

### PR DESCRIPTION
**Pull request by pugangxa**
Signed-off-by: Gang Pu <pugangxa@cn.ibm.com>

When following the instruction to debug I encountered the error:
```
Events:
  Type     Reason     Age                 From               Message
  ----     ------     ----                ----               -------
  Normal   Scheduled  2m6s                default-scheduler  Successfully assigned fybrik-system/katalog-connector-7b477c94c-675d5 to control-worker
  Normal   Pulling    44s (x4 over 2m5s)  kubelet            Pulling image "localhost:5000/fybrik-system/katalog-connector:master"
  Warning  Failed     44s (x4 over 2m5s)  kubelet            Failed to pull image "localhost:5000/fybrik-system/katalog-connector:master": rpc error: code = Unknown desc = failed to pull and unpack image "localhost:5000/fybrik-system/katalog-connector:master": failed to resolve reference "localhost:5000/fybrik-system/katalog-connector:master": failed to do request: Head "http://localhost:5000/v2/fybrik-system/katalog-connector/manifests/master": dial tcp [::1]:5000: connect: connection refused
  Warning  Failed     44s (x4 over 2m5s)  kubelet            Error: ErrImagePull
  Normal   BackOff    17s (x6 over 2m5s)  kubelet            Back-off pulling image "localhost:5000/fybrik-system/katalog-connector:master"
  Warning  Failed     6s (x7 over 2m5s)   kubelet            Error: ImagePullBackOff
```

The default tag is changed already so I must change accordingly to make it work.

### CODE REVIEW

**Comment by revit13** [comment]
hmmm, it depends on how you build the cluster: I usually do 'kind create cluster', and then using `master` tag is fine because I do not have a local registry (no images were pushed)... I do not think the README contains instructions on how to build the cluster. I agree that for local deployment `0.0.0` makes sense but maybe add a statement explaining how to build the cluster and push the images to the local registry.

**Comment by pugangxa** [comment]
> hmmm, it depends on how you build the cluster: I usually do 'kind create cluster', and then using `master` tag is fine because I do not have a local registry (no images were pushed)... I do not think the README contains instructions on how to build the cluster. I agree that for local deployment `0.0.0` makes sense but maybe add a statement explaining how to build the cluster and push the images to the local registry.

Understand and agree. Will add a statement to make it clear.

**Comment by pugangxa** [comment]
@revit13 Comments addressed, please take a look, thanks.

File: manager/README.md [hunk 1]
```diff
@@ -26,6 +43,12 @@ Follow the installation guide as usual but in the Helm installation for the cont
 helm install fybrik charts/fybrik --set global.tag=master --set manager.enabled=false -n fybrik-system --wait
 ```
 
+If your are using the the local development kind cluster please use the `0.0.0` tag:
+
```
Comment (revit13) [inline manager/README.md:47]: the the -> the

Comment (pugangxa) [reply manager/README.md:47]: Updated.

File: manager/README.md [hunk 2]
```diff
@@ -12,6 +12,23 @@ The `manager` binary includes all of the controllers that this project defines b
...
+    ../hack/setup-local-single-cluster.sh
+    ```
+    Or execute below command to setup multi-cluster environment.
+    ```
+    ../hack/setup-local-multi-cluster.sh
+    ```
+2. Use an existing cluster
+    
+    Please follow [the instructions](../pipeline/README.md) to use tekton pipeline to deploy the fybrik components to your existing cluster.
+
```
[the remark this answers is not in the document]
Comment (pugangxa) [reply manager/README.md:21]: OK, updated.

File: manager/README.md [hunk 3]
```diff
@@ -12,6 +12,13 @@ The `manager` binary includes all of the controllers that this project defines b
 
 Beyond testing, you may run and debug the manager outside the cluster using the following instructions.
 
+### Build the fybrik environment
+Follow below Instructions to build the fybrik environment.
+
+1. Install Fybrik using the [Quick Start](https://fybrik.io/v0.5/get-started/quickstart/) guide.
+
```
Comment (revit13) [inline manager/README.md:19]: Please use `dev` version:  `https://fybrik.io/dev/get-started/quickstart/`

@pugangxa I would rename `Build the fybrik environment` to `Prepare the fybrik environment` and mention that the steps until `Install control plane` (excluded) section from the quick-start needs to be executed.

File: manager/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (revit13) [reply manager/README.md:21]: Also, fybrik crd needs to be installed as well before installing fybrik:
`helm install fybrik-crd charts/fybrik-crd -n fybrik-system --wait`

[the remark this answers is not in the document]
Comment (pugangxa) [reply manager/README.md:21]: OK, mentioned that.

File: manager/README.md (hunk 3 above)
Comment (pugangxa) [reply manager/README.md:19]: OK, I saw there's already clause saying that so I just updated to make it clear.

**Review by revit13 — APPROVED** [review]

**Pull request merged**